net/curl/Portfile

```diff
@@ -167,7 +167,7 @@ if {${name} eq ${subport}} {
     }
 
     variant http2 description {Support HTTP/2 with nghttp2} {
-        depends_lib-append      port:nghttp2
+        depends_lib-append      port:libnghttp2
```

Comment:
Since this changes lib dependencies, and those get recorded in the registry, the revision needs to be increased.

Do the other ports that mention nghttp2—snort, php-swoole, html-xml-utils, apache2—need the same change?

Reply:
I applied it to all of them, and bumped their revisions.